src/System.Runtime.CompilerServices.Unsafe/pkg/System.Runtime.CompilerServices.Unsafe.pkgproj

```diff
@@ -11,7 +11,15 @@
     <!-- this package is part of the implementation closure of NETStandard.Library
          therefore it cannot reference NETStandard.Library -->
     <SuppressMetaPackage Include="NETStandard.Library" />
-    <InboxOnTargetFramework Include="uap10.0.16300" />
+
+    <!-- Since UAP and .NETCoreApp are package based we still want to enable
+         OOBing libraries that happen to overlap with their framework package.
+         This avoids us having to lock the API in our NuGet packages just 
+         to match what shipped inbox: since we can provide a new library 
+         we can update it to add API without raising the netstandard version. -->
```

Comment:
I don't fully understand this, but seems reasonable :)

Reply:
Sure.  We have the same comment elsewhere for  packages which we want to still version and grow netstandard-based-API yet be inbox on some frameworks.